app/src/main/java/max/music_cyclon/service/BeetsFetcher.java

```diff
@@ -92,6 +97,10 @@ public BeetsFetcher(String address, Resources resources) {
             for (ArrayList<Item> album : randomAlbums) {
                 items.addAll(album);
             }
+            // terrible way to remove duplicates
```

Comment:
I think this is ok and very fast way to remove duplicates. It would be cool though if you could remove the List and only use a Set.

Reply:
Yes just using the Set by default is better

Reply:
Done